cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiffTest.java

```diff
@@ -254,10 +271,10 @@ public void testChangedListenerSecurityProtocolMapFromNonDefault() {
 
     @Test
     public void testChangedMoreProperties() {
-        ArrayList<ConfigEntry> ces = new ArrayList<>();
+        ArrayList<ConfigEntry> ces = new ArrayList<>(3);
         ces.add(new ConfigEntry("inter.broker.listener.name", "david"));
         ces.add(new ConfigEntry("group.min.session.timeout.ms", "42"));
-        ces.add(new ConfigEntry("host.name", "honza"));
+        ces.add(new ConfigEntry("zookeeper.sync.time.ms", "8000"));
```

Comment:
Can we add comments for why we chose these three configs specifically?

Reply:
There is no specific reason. We are doing just 3 random changes. Property `host.name` was removed from kafka 3.0.0 config model, so we just switched to another property.

Reply:
Might be still worth a comment that they are randomly chosen without any deeper meaning ... even that is an information which might be useful.